### ISSUE TITLE
How do I run the new PHP?

### ISSUE THREAD

**Issue by benjaminBrownlee** [sstur/js2php#5]
How do I run a file of your newly compiled php? I have copied some of the converted php I generated from the online demo into a php file, but somehow it needs to reference your php methods (ex: `new Func()`).  How would I link it to these?

**Comment by benjaminBrownlee** [comment]
Never mind. I figured it out.

**Comment by sstur** [comment]
Hi @benjaminBrownlee. Thanks for checking out this project. It's an old one, from around 2014, before we had all the nice ES6+ features in JS. It does work though. As long as your JS is either written in ES5 or compiled to ES5 (which most dist builds would be, especially if targeting the browser).

> "...how good is it? Was this designed to convert large, complex blocks of code?"

I don't think this has been used in production, but it has been tested on production codebases (a few years back). Let me know if it works for you or if you run into any issues.

**Comment by benjaminBrownlee** [comment]
Thanks. I had to toy around to figure it out, but I think I have it now.  I was really excited when I saw this, but unfortunately the performance drawback is a little burdensome for my applications.  I wanted to run my own encryption algorithms server side but an essential library (bigInt) was in javascript, so I turned here.  Generating keys can be tasking in general, but this method actually caused it to be more than two hundred times slower.  I am still learning php however, so this could still be useful.
